code/game/objects/items/powerfist.dm

```diff
@@ -89,7 +89,7 @@
 		target.visible_message("<span class='danger'>[user]'s powerfist lets out a dull thunk as [user.p_they()] punch[user.p_es()] [target.name]!</span>", \
 			"<span class='userdanger'>[user]'s punches you!</span>")
 		return
-	if(gasused.total_moles() < gasperfist * fisto_setting)
+	if(gasused.total_moles() < (gasperfist * fisto_setting - (MOLAR_ACCURACY * 5)))
```

Comment:
How tested is this? Is this a fix, or a patch?

Reply:
I have been unable to reproduce the bug after this even after multiple attempts, and given that it is probably caused by QUANTIZE and floating point rounding if my understanding of it is correct, this should be sufficient.

Reply:
Could you figure out a cleaner way of checking to see if  something is within a rounding error from QUANTIZE, and turn it into a define so I can make other people use it?

Reply:
I can give it a shot if you're having a painful time of it

Reply:
This works for you?